orgs-using-d.dd

```diff
@@ -0,0 +1,61 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(DDOC_COMMENT
+    - Put your log (png/svg) to images/DORGs.
+    - max. dims: 200x100 (resize with e.g. `mogrify -resize 200x100 image.png`)
+    - if you need commas, use a paragraph `$(P ...)`
```

Comment:
No, don't use `$(P ...)` for commas here. `DORG` already generates a `$(P ...)`. You end up with `<p><p>...</p></p>` then, which is not valid HTML. You can replace commas with `$(COMMA)` instead.